supergloo/docs/mesh/tutorials/linkerd/tutorials-1-retries.md

```diff
@@ -0,0 +1,266 @@
+---
+title: "Tutorial: Configuring Retries"
+menuTitle: Retries
+description: Tutorial on how to configure Linkerd for Retries.
+weight: 3
+---
+
+### Summary
+
+In this tutorial we'll take a look at how to use SuperGloo and Linkerd to retry failed requests
+within our mesh.
+
+Prerequisites for this tutorial:
+
+- [SuperGloo Installed]({{% ref "/installation" %}})
+- [Linkerd Installed]({{% ref "/mesh/install-linkerd" %}})
+
+### Concepts
+
+**Retries**:
+
+Having a service mesh installed to your cluster allows you to automatically retry 
+failed requests. Not all requests should be retried - only requests that have an idempotent 
+result, such as most HTTP GET requests are safe to retry.
+
+By placing the task of retrying failed requests on the service mesh, your applications will no
```

Comment:
So I try to be careful about absolute-ish statements like this :) Applications may have many reasons for retrying a request, but one that they should be less concerned with is if a service call experiences some odd, intermittent network issues or  if a very small percentage of backends are throwing sporadic errors, etc. In these cases, service mesh retries are great and reduce the amount of code the developers need to worry about. There may be other reasons for their code to retry a request even if the network call succeeds. I'd maybe relax this statement a bit.